src/main/scala/no/ndla/searchapi/model/search/settings/MultiDraftSearchSettings.scala

```diff
@@ -30,5 +30,5 @@ case class MultiDraftSearchSettings(
     relevanceIds: List[String],
     statusFilter: List[ArticleStatus.Value],
     userFilter: List[String],
-    grepCodes: List[String]
+    grepCodes: List[String],
```

Comment:
Skal det være komma etter siste i lista?

Reply:
Begge deler er valid så vidt jeg vet: https://docs.scala-lang.org/sips/trailing-commas.html
Har ikke så veldig sterke meninger fra eller til selv, men jeg syns vi skal være konsekvente på det.
scalafmt har en [option](https://scalameta.org/scalafmt/docs/configuration.html#trailing-commas) for det, men tror kanskje vi kjører en litt gammel versjon siden dette ikke feiler på travis.
Burde sikkert oppdatere scalafmt i alle scala-komponentene, så kan sikkert gjøres i en egen oppgave?

Reply:
Liker faktisk litt den trailing comma med tanke på lesbarheten i github f.eks, endrer 1 linje istedet for 2. Jeg ville stemt for å skru den på i scalafmt